benchmark/cluster/README.md

```diff
@@ -0,0 +1,54 @@
+# Cluster Training Benchmark
+
+## Setup
+
+- Platform
+  - Kubernetes: v1.6.2
+  - Linux Kernel: v3.10.0
+
+- Resource
+  - CPU: 10 Cores per Pod
+  - Memory: 5GB per Pod
+
+- Docker Image
+
+  We use different base Docker Image to run the benchmark on Kubernetes:
+  - PaddlePaddle v2: paddlepaddle/paddle:latest
```

Comment:
Should use a static tag, so when `latest` tag updates, this benchmark still can be reproduced.

Reply:
Sure, but we don't have a static tag for fluid distributed training, how about a commit ID?